lib/src/components/NxButton/NxButton.tsx

```diff
@@ -16,8 +17,15 @@ const NxButton = forwardRef<HTMLButtonElement, Props>(
         [`nx-btn--${variant}`]: variant && variant !== 'secondary',
         'nx-btn--inline': inline
       });
+      return (
+        <button aria-disabled={includes('disabled', classNames) ? true : undefined}
```

Comment:
I see no harm in adding `aria-disabled="false"` so I think this can be simplified.
```suggestion
        <button aria-disabled={includes('disabled', classNames)}
```